docs/slack-apps/slack-packages.md

```diff
@@ -0,0 +1,65 @@
+---
+id: slack-packages
+title: Slack Packages
+sidebar_label: Slack Packages
+---
+
+We've built in a variety of functions to make it easier to build Dark apps with Slack. Use these functions from autocomplete (dark/slack/Slack::), the same way you would any other function.
+
+![Slack](assets/packages/slackpackage.png)
+
+### getAllConversations
+
+Takes your Slack token as a String and returns a list of Dicts containing both your public and private Slack channel names and ids.
```

Comment:
The structure of the returned Dict is not clear.

Also, this should be referred to as a Record, not a Dict (this applies to most of the other functions too).